Lombiq.Analyzers.PowerShell/Readme.md

```diff
@@ -0,0 +1,55 @@
+# Lombiq PowerShell Analyzers
```

Comment:
Don't copy the root Readme.

Reply:
![image](https://user-images.githubusercontent.com/4007293/178172004-283ce7cb-9149-4f3a-be95-f0f8b42eea18.png)